### PR TITLE
sql: explain cop capabilities

### PR DESCRIPTION
**Pull request by CaitinChen**
Via https://github.com/pingcap/docs/pull/850
@lilin90 @zz-jason PTAL @_@

### CODE REVIEW

File: sql/understanding-the-query-execution-plan.md [hunk 1]
```diff
@@ -79,7 +79,9 @@ mysql> EXPLAIN SELECT count(*) FROM trips WHERE start_date BETWEEN '2017-07-01 0
 
 ### Task 简介
 
-目前 TiDB 的计算任务隶属于两种不同的 task：cop task 和 root task。cop task 是指被下推到 KV 端分布式执行的计算任务，root task 是指在 TiDB 端单点执行的计算任务。SQL 优化的目标之一是将计算尽可能的下推到 KV 端执行。
+目前 TiDB 的计算任务隶属于两种不同的 task：cop task 和 root task。cop task 是指使用 TiKV 中的 Coprocessor 执行的计算任务，root task 是指在 TiDB 中执行的计算任务。
```
Comment (zz-jason) [inline sql/understanding-the-query-execution-plan.md]:
```suggestion
目前 TiDB 的计算任务隶属于两种不同的 task：cop task 和 root task。cop task 是指使用 TiKV Coprocessor 中执行的计算任务，root task 是指在 TiDB 中执行的计算任务。
```

File: sql/understanding-the-query-execution-plan.md [hunk 2]
```diff
@@ -79,7 +79,9 @@ mysql> EXPLAIN SELECT count(*) FROM trips WHERE start_date BETWEEN '2017-07-01 0
...
+
+SQL 优化的目标之一是将计算尽可能地下推到 TiKV 中执行。Coprocessor 能协助执行 SQL 函数（包括聚合函数和标量函数）、SQL `LIMIT` 操作、索引扫描和表扫描。但是，所有的 Join 操作都作为 root task 执行。
```
Comment (zz-jason) [inline sql/understanding-the-query-execution-plan.md]:
```suggestion
SQL 优化的目标之一是将计算尽可能地下推到 TiKV 中执行。Coprocessor 能支持大部分 SQL 内建函数（包括聚合函数和标量函数）、SQL `LIMIT` 操作、索引扫描和表扫描。但是，所有的 Join 操作都只能作为 root task 在 TiDB 上执行。
```

**Comment by CaitinChen** [comment]
@lilin90 PTAL @zz-jason PTAL again. 0_0

File: sql/understanding-the-query-execution-plan.md [hunk 3]
```diff
@@ -79,7 +79,9 @@ mysql> EXPLAIN SELECT count(*) FROM trips WHERE start_date BETWEEN '2017-07-01 0
 
 ### Task 简介
 
-目前 TiDB 的计算任务隶属于两种不同的 task：cop task 和 root task。cop task 是指被下推到 KV 端分布式执行的计算任务，root task 是指在 TiDB 端单点执行的计算任务。SQL 优化的目标之一是将计算尽可能的下推到 KV 端执行。
+目前 TiDB 的计算任务隶属于两种不同的 task：cop task 和 root task。cop task 是指使用 TiKV Coprocessor 中执行的计算任务，root task 是指在 TiDB 中执行的计算任务。
```
Comment (lilin90) [inline sql/understanding-the-query-execution-plan.md]:
```suggestion
目前 TiDB 的计算任务隶属于两种不同的 task：cop task 和 root task。cop task 是指使用 TiKV 中的 Coprocessor 执行的计算任务，root task 是指在 TiDB 中执行的计算任务。
```

**Pull request merged**